### PR TITLE
Fix example syntax errors

### PR DESCRIPTION
**Pull request by rubiesonthesky**
#### Summary

Fixing some incorrect language markings and some js syntax errors in example codes.

#### Motivation

Will make it easier to use tools later for checking the example codes and use correct syntax highlighting in the web page.

#### Metadata

- [ ] Adds a new document
- [ ] Rewrites (or significantly expands) a document
- [x] Fixes a typo, bug, or other error

### CODE REVIEW

File: files/en-us/web/javascript/reference/global_objects/function/caller/index.md [hunk 1]
```diff
@@ -30,7 +30,7 @@ f(2);
 
 At the moment `stop()` is called the call stack will be:
 
-```js
+```text
```
Comment (lionralfs) [inline files/en-us/web/javascript/reference/global_objects/function/caller/index.md:33]: I've found no usages of `text`, let's use `plain`

```suggestion
```plain
```

File: files/en-us/web/security/secure_contexts/index.md [hunk 2]
```diff
@@ -41,7 +41,7 @@ Pages can use feature detection to check whether they are in a secure context or
 if (window.isSecureContext) {
   // Page is a secure context so service workers are now available
   navigator.serviceWorker.register("/offline-worker.js").then(function () {
-    ...
+    // callback logic
```
Comment (lionralfs) [inline files/en-us/web/security/secure_contexts/index.md:44]: In the other syntax-fixing PRs, we've been doing:

```suggestion
    // …
```

File: files/en-us/webassembly/exported_functions/index.md [hunk 3]
```diff
@@ -63,7 +63,7 @@ typeof testFunc;
 
 you'll get the result `function` returned. You can then go on to do pretty much anything to this function that you can do to other [functions](/en-US/docs/Web/JavaScript/Reference/Global_Objects/Function) in JavaScript — [`call()`](/en-US/docs/Web/JavaScript/Reference/Global_Objects/Function/call), [`bind()`](/en-US/docs/Web/JavaScript/Reference/Global_Objects/Function/bind), etc. `testFunc.toString()` returns an interesting result:
 
-```js
+```text
```
Comment (lionralfs) [inline files/en-us/webassembly/exported_functions/index.md:66]:
```suggestion
```plain
```

File: files/en-us/web/javascript/reference/statements/export/index.md [hunk 4]
```diff
@@ -253,14 +253,14 @@ This is what it would look like using code snippets:
...
+const myFunction = () => { /* function body */ };
+const myVariable = "..."; // assign something useful to myVariable
 export { myFunction, myVariable };
 ```
 
 ```js
 // In childModule2.js
-const myClass = ...; // assign something useful to myClass
+const myClass = class ClassName { /* ... */ };
 export { myClass };
```
Comment (Josh-Cena) [inline files/en-us/web/javascript/reference/statements/export/index.md:264]: These are being fixed in https://github.com/mdn/content/pull/18761

**Review by Josh-Cena — APPROVED** [review]
Thank you!

**Pull request merged**